boards/arduino/portenta_h7/arduino_portenta_h7-common.dtsi

```diff
@@ -196,3 +196,44 @@ zephyr_udc0: &usbotg_fs {
 		};
 	};
 };
+
+&quadspi {
+	pinctrl-0 = < &quadspi_bk1_io0_pd11
+				  &quadspi_bk1_io1_pd12
+				  &quadspi_bk1_io2_pf7
+				  &quadspi_bk1_io3_pd13
+				  &quadspi_bk1_ncs_pg6
+				  &quadspi_clk_pf10 >;
+	pinctrl-names = "default";
+	status = "okay";
+
+	tc58cv: tc58vg0s3hraig@0 {
+		compatible = "st,stm32-qspi-nor";
+		reg = < 0 >;
+		qspi-max-frequency = < 40000000 >;
+		size = < DT_SIZE_M(128) >;
```

Comment:
Here, there is now a new usage of reg property to encode the actual memory address mapping to be taken into account.
See https://github.com/zephyrproject-rtos/zephyr/pull/68274
```
		reg = <0x90000000 DT_SIZE_M(16)>; /* 128 Mbits */
```

Note also the unit that might need to be fixed. I let you check this part